### PR TITLE
add delete restore points API

### PR DESCRIPTION
**Pull request by sarvesh-s**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by jaredmoo** [comment]
> The title of the PR is clear and informative.
I don't think this is done yet :) can you edit title to summarize the change?

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/restorePoints.json [hunk 1]
```diff
@@ -173,6 +173,50 @@
...
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
```
Comment (jaredmoo) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/restorePoints.json]: is delete long-running, i.e. returns 202 + operationresults? Or does it complete within <5 seconds?

**Review by jaredmoo — CHANGES_REQUESTED** [review]
Please change title and check if delete is long-running

**Comment by sarvesh-s** [comment]
fixed the title

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/restorePoints.json (hunk 1 above)
Comment (jaredmoo) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/restorePoints.json]: Cool, just checking we do not need to worry about more timeouts :)

Comment (sarvesh-s) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/restorePoints.json]: so yes, <5 seconds

**Review by jaredmoo — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
The title says "change to async for create API" but I don't see any changes to the create API.  Is that still in progress?

**Comment by sarvesh-s** [comment]
Actually it was already pushed to main branch, I will correct the title :)

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**